ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -653,12 +666,136 @@ export const RuntimePanel = fp.flow(
     return <Spinner style={{width: '100%', marginTop: '5rem'}}/>;
   }
 
+  const createRuntimeRequest = (runtime: RuntimeConfig) => {
+    const runtimeRequest: Runtime = runtime.computeType === ComputeType.Dataproc ? {
+      dataprocConfig: {
+        ...runtime.dataprocConfig,
+        masterMachineType: runtime.machine.name,
+        masterDiskSize: runtime.diskSize
+      }
+    } : runtime.computeType === ComputeType.Standard ? {
+      gceConfig: {
+        machineType: runtime.machine.name,
+        diskSize: runtime.diskSize
+      }
+    } : null;
+
+    // If the selected runtime matches a preset, plumb through the appropriate configuration type.
+    runtimeRequest.configurationType = fp.get(
+      'runtimeTemplate.configurationType',
+      fp.find(
+        ({runtimeTemplate}) => presetEquals(runtimeRequest, runtimeTemplate),
+        runtimePresets)
+    ) || RuntimeConfigurationType.UserOverride;
+
+    return runtimeRequest;
+  };
+
+  const renderUpdateButton = () => {
+    return <Button
+      aria-label='Update'
+      disabled={
+        !runtimeChanged
+        // Casting to RuntimeStatus here because it can't easily be done at the destructuring level
+        // where we get 'status' from
+        || ![RuntimeStatus.Running, RuntimeStatus.Stopped].includes(status as RuntimeStatus)
+      }
+      onClick={() => {
+        setRequestedRuntime(createRuntimeRequest(newRuntimeConfig));
+        onUpdate();
+      }}>
+      Update
+    </Button>;
+  };
+
+  const renderCreateButton = () => {
+    return <Button
+      aria-label='Create'
+      onClick={() => {
+        setRequestedRuntime(createRuntimeRequest(newRuntimeConfig));
+      }}>
```

Comment:
The create button from the Create panel is supposed to close the panel (by invoking `onUpdate`), may have been lost in the merge conflicts